sdk/python/jobs/pipelines/pipeline_with_components_use_primitive_output/command_component_with_primitive_output.yaml

```diff
@@ -0,0 +1,37 @@
+$schema: https://azuremlschemas.azureedge.net/latest/commandComponent.schema.json
```

Comment:
we should follow the existing naming pattern and add ci check
![image](https://github.com/Azure/azureml-examples/assets/47586720/2fd2d7f1-3278-4ab1-8c8f-080401d3d55d)